notebooks/ldb-eda-structure.ipynb

```diff
@@ -0,0 +1,1941 @@
+{
```

Comment:
<p>Both head() and describe() methods are pretty important for the analysis, but I think it would be nice if you put some markdowns to briefly explain it, so that anyone that reads the notebook can understand why you did it like this and what are the conclusions you got.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Sure. I gave the same feedback for PYC, I will make my comments and conclusions in our next job "data cleaning".